### PR TITLE
Snackbars - 31-ods component snackbars new - retrieving the snackbar branch with a new commit

### PR DESCRIPTION
**Pull request by Tayebsed93**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Types of change

- Bug fix (non-breaking which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Refactoring (non-breaking change)
- Breaking change (fix or feature that would change existing functionality)

#### Accessibility

- [ ] My change follows accessibility good practices

#### Design

- [ ] My change respects the design guidelines defined in [Orange Design System](https://system.design.orange.com/0c1af118d/p/8118d1-web)
- [ ] My change is compatible with responsive display

#### Documentation

- [ ] My change introduces changes to the documentation and/or I have updated the documentation accordingly

### Checklist (for Core Team only)

- [ ] Manually test on simulators and on a physical device (Android & iOS, orientation change, dark/light mode)
- [ ] Code review
- [ ] Design review
- [ ] A11y review

### CODE REVIEW

File: app/lib/ui/components/snackbars/snackbars.dart [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Padding(
+            padding: EdgeInsets.all(spacingM),
+            child: Text(
+              'Customize the snackbar before displaying it',
```
Comment (ludovic35) [inline app/lib/ui/components/snackbars/snackbars.dart:56]: Should be localized

File: app/lib/ui/components/snackbars/snackbars.dart [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+                      ? () {}
+                      : null,
+                );
+              }
+              if (customizationState?.hasLongerAction == true) {
+                OdsSnackbars.showSnackbarLongerAction(
+                  context: context,
+                  content: AppLocalizations.of(context)!
+                      .componentSnackbarsTwoLineLongerActionText,
+                  label: 'Longer action',
```
Comment (ludovic35) [inline app/lib/ui/components/snackbars/snackbars.dart:112]: Should be localized

File: app/lib/ui/components/snackbars/snackbars.dart [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+              style: Theme.of(context).textTheme.headlineSmall,
+            ),
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class _CustomizationContent extends StatelessWidget {
```
Comment (ludovic35) [inline app/lib/ui/components/snackbars/snackbars.dart:128]: I made a test:
The text is on tow lines, but two  line switch is off : 
image

**Review by ludovic35 — CHANGES_REQUESTED** [review]

File: app/lib/ui/components/snackbars/snackbars.dart (hunk 3 above)
Comment (Tayebsed93) [reply app/lib/ui/components/snackbars/snackbars.dart:128]: on merge puis on voit avec les designer. A corriger par la suite avec un ticket de bug si besoin

**Review by ludovic35 — APPROVED** [review]

**Pull request merged**